packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -615,6 +630,10 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
         }
     }
 
+    private isElectron(): boolean {
```

Comment:
Either mark this protected or directly use `environment.electron.is()`. It doesn't really add value here.